Content.Shared/Damage/Components/DamageableComponent.cs

```diff
@@ -65,6 +65,13 @@ public sealed partial class DamageableComponent : Component
         [DataField("radiationDamageTypes")]
         public List<ProtoId<DamageTypePrototype>> RadiationDamageTypeIDs = new() { "Radiation" };
 
+        /// <summary>
+        ///     Group types that affect the pain overlay.
+        /// </summary>
+        ///     TODO: Add support for adding damage types specifically rather than whole damage groups
+        [DataField]
+        public List<ProtoId<DamageGroupPrototype>> PainDamageGroupIDs = new() { "Brute", "Burn" };
```

Comment:
actually i missed it, you still have to have the other name for the datafield, it's more convinient i suppose 
```suggestion
        [DataField("painDamageGroups")]
        public List<ProtoId<DamageGroupPrototype>> PainDamageGroupIDs = new() { "Brute", "Burn" };
```

Reply:
Added it back in!